libraries/node-core-library/src/Terminal/Terminal.ts

```diff
@@ -0,0 +1,281 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { EOL } from 'os';
```

Comment:
This should probably happen at the provider level.  If we're writing to a log file, we need deterministic output.

Reply:
In fact we possibly should normalize this somehow, in case `write()` was called with mixed newlines.

---
In reply to: [227609313](https://github.com/Microsoft/web-build-tools/pull/898#discussion_r227609313) [](ancestors = 227609313)

Reply:
Let's do that separately.

---
In reply to: [227609496](https://github.com/Microsoft/web-build-tools/pull/898#discussion_r227609496) [](ancestors = 227609496,227609313)